VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -564,29 +564,32 @@ public Properties trackOpenInBrowser(String url) {
     public Properties trackTranscriptLanguage(String videoId,
             Double currentTime, String lang, String courseId, String unitUrl) {
         try{
-            VideoEvent vEvent = getCommonPropertiesWithCurrentTime(currentTime,
+            SegmentAnalyticsEvent aEvent = getCommonPropertiesWithCurrentTime(currentTime,
                     videoId, Values.TRANSCRIPT_LANGUAGE);
-            vEvent.properties.putValue(Keys.LANGUAGE, lang);
-            vEvent.setContext(courseId, unitUrl,
+            aEvent.properties.putValue(Keys.LANGUAGE, lang);
+            aEvent.setContext(courseId, unitUrl,
                     Values.VIDEOPLAYER);
 
-            tracker.track(Keys.LANGUAGE_CLICKED, vEvent.properties);
-            return vEvent.properties;
+            tracker.track(Keys.LANGUAGE_CLICKED, aEvent.properties);
+            return aEvent.properties;
         }catch(Exception e){
             logger.error(e);
         }
         return null;
     }
 
+    /**
+     * This function is used to track if user clicks on Sign up on landing page
+     */
     @Override
     public Properties trackUserDoesNotHaveAccount() {
```

Comment:
Nitpick: Can we rename this to `trackUserSignUpForAccount`?  `USER_NO_ACCOUNT` does not imply that the user actually clicked on the sign-up button.